app/scripts/app.js

```diff
@@ -174,6 +174,11 @@ angular
         templateUrl: 'views/edit/deployment-config.html',
         controller: 'EditDeploymentConfigController'
       })
+      .when('/project/:project/browse/stateful-sets/:statefulset', {
```

Comment:
Done, though the annoying thing about camelCase here is that it doesn't work in the controller when using `$routeParams`. It will still be `$routeParams.statefulset` :(